test/framework/clusterctl/client.go

```diff
@@ -230,6 +231,13 @@ func ConfigCluster(ctx context.Context, input ConfigClusterInput) []byte {
 		TargetNamespace:          input.Namespace,
 	}
 
+	outputPath := filepath.Join(filepath.Dir(input.ClusterctlConfigPath), fmt.Sprintf("clusterctl-upgrade-config-%s.yaml", input.ClusterName))
+	copyAndAmendClusterctlConfig(ctx, copyAndAmendClusterctlConfigInput{
```

Comment:
Can you please drop the if in l.135 (Just l.135+l.143)? I think it's fine to always copy, just want to do it consistently in both cases